### PR TITLE
Feat admin login

### PR DESCRIPTION
**Pull request by anchitjulaniya**

### CODE REVIEW

File: src/Components/atoms/Input/Input.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  type?: string;
+  placeholder?: string;
+  value: string;
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+  name?: string;
+  label?: string;
+  outerBoxclassName?: string;
+  inputBoxclassName?: string;
+  labelBoxclassName?: string;
+}
```
Comment (drumil32) [inline src/Components/atoms/Input/Input.tsx:13]: create folder with name types and put this things there

File: src/Pages/Login/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+          outerBoxclassName="relative mt-2 w-full"
+          inputBoxclassName="border-1 peer block w-full appearance-none rounded-lg border border-gray-300 bg-transparent px-2.5 pt-4 pb-2.5 text-sm text-gray-900 focus:border-[#50B500] focus:outline-none focus:ring-0"
+          labelBoxclassName="origin-[0] peer-placeholder-shown:top-1/2 peer-placeholder-shown:-translate-y-1/2 peer-placeholder-shown:scale-100 peer-focus:top-2 peer-focus:-translate-y-4 peer-focus:scale-75 peer-focus:px-2 peer-focus:text-[#50B500] absolute left-1 top-2 z-10 -translate-y-4 scale-75 transform cursor-text select-none bg-white px-2 text-sm text-gray-500 duration-300"
+        />
+        <Input
+          type="password"
+          value={formData.password}
+          onChange={handlePasswordChange}
+          name="password"
+          placeholder=" "
```
Comment (drumil32) [inline src/Pages/Login/Login.tsx:95]: remove placeholder if its not required. you can make it optional.

File: README.md [hunk 3]
```diff
@@ -1 +1 @@
-# Raithan-AdminPanel
+#Raithan-AdminPanel
```
Comment (drumil32) [inline README.md:1]: why you removed space from here?

**Comment by anchitjulaniya** [comment]
i have updated code

File: src/Components/atoms/Input/Input.tsx (hunk 1 above)
Comment (anchitjulaniya) [reply src/Components/atoms/Input/Input.tsx]: done

File: src/Pages/Login/Login.tsx (hunk 2 above)
Comment (anchitjulaniya) [reply src/Pages/Login/Login.tsx]: done

**Review by drumil1602 — APPROVED** [review]

**Pull request merged**